samples/samples.dpr

```diff
@@ -6,10 +6,19 @@ uses
   Horse,
   Horse.OctetStream,
   System.Classes,
-  System.SysUtils;
+  System.SysUtils,
+  Winapi.Windows;
 
 {$R *.res}
 
+function GetPath: string;
```

Comment:
You cannot use the ExtractFilePath(ParamStr(0)) ?

Reply:
Yes, in ISAPI ParamStr (0) will not work. If it's not a problem, it's easy to switch.